### PR TITLE
[FrameworkBundle] deal with explicitly enabled workflow nodes

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | not yet
| Fixed tickets | #28662
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/xml/workflows_explicitly_enabled.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:framework="http://symfony.com/schema/dic/symfony"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd
+                        http://symfony.com/schema/dic/symfony http://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+    <framework:config>
+        <framework:workflow enabled="true" name="foo" type="workflow" initial-place="bar">
```
[the remark this answers is not in the document]
Comment (xabbuh) [reply src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/xml/workflows_explicitly_enabled.xml]: @symfony/deciders What is your opinion here? :)

[the remark this answers is not in the document]
Comment (nicolas-grekas) [reply src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/xml/workflows_explicitly_enabled.xml]: actually what's the meaning of the "enabled" option on the configuration?
as soon as one workflow is defined, workflow is enable - and if no workflow are defined, it's disable.
Wouldn't this make sense?

[the remark this answers is not in the document]
Comment (xabbuh) [reply src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/xml/workflows_explicitly_enabled.xml]: Generally yes, this option was introduced to be able to explicitly enable the workflow services while not using the config to define your workflows (see #24051 for the context).

[the remark this answers is not in the document]
Comment (lyrixx) [reply src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/xml/workflows_explicitly_enabled.xml]: XML are boring for the framework configuration.

I would like instead close #28662 as won't fix. Use should use the "regular" way to define workflow.

[the remark this answers is not in the document]
Comment (lyrixx) [reply src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/xml/workflows_explicitly_enabled.xml]: Anyway, I would go for supporting 
`<framework:workflow enabled="true" />`
or your current exemple without `enabled="true"` 
and that's all

**Comment by xabbuh** [comment]
The extension now throws when `enabled="false` is present for a single workflow.

Status: Needs Review

**Review by lyrixx — APPROVED** [review]

**Comment by lyrixx** [comment]
... and one more bug fixed, thank you Christian.

**Pull request merged**